pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -201,7 +201,22 @@ class CollectionViewSet(
     filterset_class = CollectionFilter
     pagination_class = LimitOffsetPagination
 
-    DEFAULT_ACCESS_POLICY = _PERMISSIVE_ACCESS_POLICY
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": "destroy",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:ansible.delete_collection",
+            },
+            {
+                "action": ["update", "partial_update"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:ansible.change_collection",
```

Comment:
we shouldn't be using the collection version or collection permissions directly, since we don't ever track permissions on content. Instead permissions to perform operations on content in the repo should be based off of repository level permissions. For now you could probably use "modify repo content" as a proxy for this, but it may be worth it to create some more granular permissions such as "modify repo collections", "modify repo namespaces" etc.